dexs/momentum/index.ts

```diff
@@ -0,0 +1,76 @@
+import { postURL} from "../../utils/fetchURL"
+import { Chain } from "@defillama/sdk/build/general";
+import { FetchResultV2, SimpleAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import { getUniqStartOfTodayTimestamp } from "../../helpers/getUniSubgraphVolume";
+
+type IUrl = {
+    [s: string]: string;
+}
+
+const url: IUrl = {
+    [CHAIN.SUI]: `https://app.sentio.xyz/api/v1/insights/mmt-finance/clmm-dashboard/query`
+}
+
+const options = {
+    headers: {
+        'Content-Type': 'application/json',
+        'api-key': 'sd0mYLVwi9gZx8l0FHryM5pQY5VEbU8RX',
+    },
+};
+
+const fetch = (chain: Chain) => {
+    return async ({endTimestamp }): Promise<FetchResultV2> => {
+        const dayTimestamp = getUniqStartOfTodayTimestamp(new Date(endTimestamp * 1000));
+        const data = {
+            timeRange: {
+                start: 'now-1d',
```

Comment:
how can we pass `fromTimestamp` & `toTimestamp` here? else, it will only return recent 24h data, we wont be able to pull historical data

for now, we have this adapter, but it needs to be replaced: https://github.com/DefiLlama/dimension-adapters/blob/master/fees/momentum.ts#L7